node/actors/network/src/gossip/mod.rs

```diff
@@ -173,57 +165,4 @@ impl Network {
         })
         .await;
     }
-
-    /// Task that reacts to new votes being added and looks for an L1 batch QC.
-    /// It persists the certificate once the quorum threshold is passed.
-    pub(crate) async fn run_batch_qc_finder(&self, ctx: &ctx::Ctx) -> ctx::Result<()> {
-        let Some(attesters) = self.genesis().attesters.as_ref() else {
-            tracing::info!("no attesters in genesis, not looking for batch QCs");
-            return Ok(());
-        };
-        let genesis = self.genesis().hash();
-
-        let mut recv_votes = self.batch_votes.subscribe();
-        let mut recv_status = self.attestation_status.subscribe();
-
-        // Subscribe starts as seen but we don't want to miss the first item.
-        recv_status.mark_changed();
-
-        loop {
-            async {
-                // Wait until the status indicates that we're ready to sign the next batch.
-                let Some(batch_number) = sync::changed(ctx, &mut recv_status)
-                    .instrument(tracing::info_span!("wait_for_attestation_status"))
-                    .await?
-                    .next_batch_to_attest
-                else {
-                    return Ok(());
```

Comment:
Oops, this should have been `continue` 🤔

Reply:
:man_shrugging: